### DISCUSSION TITLE
Custom domain into institutional domain

### DISCUSSION THREAD

**Discussion by UE2lab** [community/community#61799] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello everyone! I am trying to change my Github pages domain into become institutional domain (ex. ~example.ac.kr). Is that possible? 

when I inputted the web domain (~example.ac.kr), it is showing warning like this;

(~example.ac.kr) is improperly configured
Domain's DNS record could not be retrieved. For more information, see [documentation](https://docs.github.com/articles/setting-up-a-custom-domain-with-github-pages/) (InvalidDNSError).

Could you please help me figure it out? Thanks!

**Comment by nirav-gajera** [comment]
https://stackoverflow.com/questions/54059217/how-to-fix-domain-does-not-resolve-to-the-github-pages-server-error-in-github

https://stackoverflow.com/questions/57680965/how-to-resolve-domains-dns-record-could-not-be-retrieved-in-github-page-creat

https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/troubleshooting-custom-domains-and-github-pages
